TodoApi/appsettings.Development.json

```diff
@@ -1,24 +1,52 @@
 {
-  "Serilog": {
-    "MinimumLevel": {
-      "Default": "Information",
-      "Override": {
-        "Microsoft": "Information",
-        "Microsoft.Hosting.Lifetime": "Information"
-      }
+    "Serilog": {
+        "Using": [ "Serilog.Sinks.Seq", "Serilog.Sinks.Elasticsearch" ],
+        "MinimumLevel": {
+            "Override": {
+                "Microsoft": "Information"
+            }
+        },
+        // Available Sinks (aka. Destinations to recieve logging data): https://github.com/serilog/serilog/wiki/Provided-Sinks
+        "WriteTo": [
+            {
+                "Name": "Console",
+                "Args": {
+                    "OutputTemplate": "{Timestamp:yyyy-MM-dd HH:mm:ss.fff} {Level} - {Message:lj}{NewLine}{Exception}"
+                }
+            },
+            // REF: https://github.com/datalust/serilog-sinks-seq
+            {
+                "Name": "Seq",
+                "Args": {
+                    "serverUrl": "http://localhost:8081",
+                    "restrictedToMinimumLevel": "Information"
+                }
+            },
+            // REF: https://github.com/serilog-contrib/serilog-sinks-elasticsearch
+            {
+                "Name": "Elasticsearch",
+                "Args": {
+                    "NodeUris": "http://localhost:9200",
```

Comment:
What happens if elastic search isn't running?

Reply:
I'm assuming the same as with Seq. See comments above.

Reply:
🔧